lib/custom_widgets/genres.dart

```diff
@@ -17,8 +18,8 @@ class WidgetedGenres extends StatelessWidget {
           (e) => Padding(
             padding: const EdgeInsets.only(left: genresWordLeftPadding),
             child: Text(
-              e,
-              style: MovieTextStyles.genresTextStyle,
+              e.toString(),
```

Comment:
Could you use a more descriptive name please

Reply:
Done